pkg/engine/harpoon.go

```diff
@@ -299,13 +300,13 @@ func (hc *HarpoonConfig) processRaw(ctx context.Context, target *api.Target, sch
 		}
 		targetFile, err = hc.applyInitial(ctx, mo, fileName, target.Raw.TargetPath, &tag, subDirTree)
 		if err != nil {
-			log.Fatalf("Repo: %s Method: %s, error while processing the repository: %s", target.Name, rawMethod, err)
+			checkInitialErr(target.Name, rawMethod, err)
```

Comment:
`Fatalf` calls `os.Exit` but checkInitialErr does not. Is the idea here that the code should not exit on these types of errors?

Reply:
Yes